pytest/tests/loadtest/locust/common/congestion.py

```diff
@@ -0,0 +1,122 @@
+import json
+import pathlib
+import sys
+
+from locust import events, runners
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / "lib"))
+
+import account
+import cluster
+import common.base as base
+import key
+import transaction
+
+
+class ComputeSha256(base.Transaction):
+    """Transaction with a large input size."""
+
+    def __init__(
+        self,
+        contract_account_id: str,
+        sender: base.Account,
+        size_bytes: int,
+    ):
+        super().__init__()
+        self.contract_account_id = contract_account_id
+        self.sender = sender
+        self.size_bytes = size_bytes
+
+    def sign_and_serialize(self, block_hash) -> bytes:
+        return transaction.sign_function_call_tx(
+            self.sender.key,
+            self.contract_account_id,
+            "ext_sha256",
+            json.dumps(["a" * self.size_bytes]).encode("utf-8"),
+            300 * account.TGAS,
+            # Attach exactly 1 yoctoNEAR according to NEP-141 to avoid
+            # calls from restricted access keys.
+            1,
```

Comment:
Wouldn't the congestion contract work without balance?

Reply:
Good catch, I've tested and it works!